pkg/resource/logging-agents-toggle/observability_bundle_configmap.go

```diff
@@ -19,16 +19,19 @@ type app struct {
 }
 
 // GenerateObservabilityBundleConfigMap returns a configmap for
-// the observabilitybundle application to enable promtail.
+// the observabilitybundle application to enable logging agents.
 func GenerateObservabilityBundleConfigMap(lc loggedcluster.Interface, observabilityBundleVersion semver.Version) (v1.ConfigMap, error) {
-	appName := "promtail"
+	promtailAppName := "promtail"
 	if observabilityBundleVersion.LT(semver.MustParse("1.0.0")) {
-		appName = "promtail-app"
+		promtailAppName = "promtail-app"
 	}
 
 	values := Values{
 		Apps: map[string]app{
-			appName: {
+			promtailAppName: {
+				Enabled: true,
+			},
+			"grafanaAgent": {
```

Comment:
We need to use an observability-bundle version toggle here

Reply:
https://gigantic.slack.com/archives/C01176DKNP4/p1699452290447569

Reply:
I did it, it is coming...